### PR TITLE
Chore: some quick fixes and removal of redundant css from homepage

### PR DESCRIPTION
**Pull request by RiyaMathew-11**
## What type of PR is this? 

- [x] ⚙️Chore
- [x] 🐛Bug Fix

## Description
This is a small PR that handles 3 things: 

1. A bug fix: The current home page shows redundancy in content for goal text.
2. Chore: Added fields to showcase more programs in homepage - currently had only three - made it 6 for now. 

   -  Six seemed the ideal number for home page, can add more in future or redirect to a separate page for programs listing.
3. Removes a whole lot of unwanted CSS from homepage.
4. Some font size and alignment fixes in events' templates.

## Checklist
- [x] I have read the [contribution guidelines]("./docs/CONTRIBUTING.md").
- [x] I have tested these changes locally.
- [x] I have updated the documentation (If applicable).
- [x] The code follows the project's coding standards.
- [x] I have added/updated tests, if applicable.

## Screenshots/GIFs/Screen Recordings 

Screenshot 2024-03-27 at 1 09 40 AM

Screenshot 2024-03-20 at 9 30 21 PM

Screenshot 2024-03-27 at 8 34 43 PM

### CODE REVIEW

File: fossunited/public/css/custom.css [hunk 1]
```diff
@@ -330,7 +330,7 @@ h6{
 
 .primary-button{
     display: flex;
-    padding: 0.625rem 0.75rem;
```
Comment (harshtandiya) [inline fossunited/public/css/custom.css:333]: Why? These are the dimensions which are in design. Please attach a screenshot for before and after of this.

File: fossunited/fossunited/web_template/home_page___events/home_page___events.html [hunk 2]
```diff
@@ -312,25 +133,4 @@ <h2 class="flagship-card-heading">FOSS Hack</h2>
...
-        {% set events = frappe.get_all("FOSS Chapter Event", fields=["event_name", "banner_image", "route", "must_attend", "event_location", "map_link", "event_start_date"], filters={"status": "Approved", "event_start_date": ['>=', frappe.utils.now()]}, page_length=6, order_by='event_start_date') %}
-        <div class="events-grid-4">
-            {% for event in events %}
-                {{ event_card(event) }}
-            {% endfor %}
-        </div>
-
-
-
-    </div>
```
Comment (harshtandiya) [inline fossunited/fossunited/web_template/home_page___events/home_page___events.html:335]: Why was this removed?

**Review by harshtandiya — CHANGES_REQUESTED** [review]

File: fossunited/public/css/custom.css (hunk 1 above)
Comment (RiyaMathew-11) [reply fossunited/public/css/custom.css:333]: I felt the buttons were a bit constricted in the screen - 0.75:1 gives it a 3:4 proportion - which when I checked gives a bit of expansion and keeps proportion but does not change much of size.

Before Change:  

Screenshot 2024-03-27 at 7 56 22 PM

After change: 

Screenshot 2024-03-27 at 7 57 07 PM

File: fossunited/fossunited/web_template/home_page___events/home_page___events.html (hunk 2 above)
Comment (RiyaMathew-11) [reply fossunited/fossunited/web_template/home_page___events/home_page___events.html:335]: There is an [upcoming events template](https://github.com/fossunited/fossunited/tree/develop/fossunited/fossunited/web_template/foss_events___upcoming_events_section) that is used in the events page. 

We could reuse the same in home page - having both of them is redundant. The alignment issue wrt container in this template - I've fixed it in a commit.

**Review by harshtandiya — APPROVED** [review]

**Pull request merged**